winpr/libwinpr/utils/asn1/asn1.c

```diff
@@ -742,9 +742,9 @@ size_t WinPrAsn1EncContextualOID(WinPrAsn1Encoder* enc, WinPrAsn1_tagId tagId,
 }
 
 size_t WinPrAsn1EncContextualOctetString(WinPrAsn1Encoder* enc, WinPrAsn1_tagId tagId,
-                                         const WinPrAsn1_OctetString* octets)
+                                         const WinPrAsn1_OctetString* oid)
 {
```

Comment:
This renaming is unfortunate, octets strings != oid

Reply:
ok, renamed.